rest-api/src/main/resources/paths/studies/v5_studies_studyId_exporter3.yml

```diff
@@ -0,0 +1,20 @@
+post:
+    operationId: initExporter3ForStudy
+    summary: Initializes configs and Synapse resources for Exporter 3.0.
+    tags:
+        - Studies
+        - _For Study Coordinators
```

Comment:
Study Coordinator isn't included in the server code. Not sure if it should be removed here or added to the controller.

Reply:
I copied the list from v5_studies_studyId.yml, which includes For Study Coordinators, but apparently that Controller method doesn't accept Study Controllers either. I'll remove it from this file (but not the existing file so I don't break anything).